lib/venue.rb

```diff
@@ -1,2 +1,19 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+
+  def initialize (venue, capacity)
+    @name = venue
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patrons)
+    @patrons = ['Mike', 'Megan', 'Bob']
```

Comment:
What data structure is `@patrons`? How can you add something to an array? 

As it is now, you have initialized `@patrons` as an empty array. Then on line 11 you are hard coding it. What if there are more patrons to be added?

https://ruby-doc.org/core-2.5.0/Array.html#:~:text=Adding%20Items%20to,4%2C%205%2C%206%5D